### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about the Capstone project. The following requirements have been done:
- Make sure that there are no linter errors. :white_check_mark: 
- Make sure that you used the correct GitHub flow. :white_check_mark: 
- Make sure that you documented your work in a professional way. :white_check_mark: 
- Follow our list of best practices for HTML & CSS. :white_check_mark: 
- Follow our list of best practices for JavaScript. :white_check_mark: 
- personalize the content of your page. Choose a topic that is different than the one in the original design. :white_check_mark: 
- build only these 2 pages :white_check_mark: 
- Each of these pages should have versions for 2 different screen sizes :white_check_mark: 
- user interactions :white_check_mark: 
- Dynamic page :white_check_mark: 
- Loom video for the project : https://www.loom.com/share/c79b5ffbd1c047bc99a31bee71da2ce3 :white_check_mark:

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                            </a>
+                        </div>
+                        <div class="col-sm">
+                            <ul class="nav justify-content-center">
+                                <li class="nav-item">
+                                  <a class="nav-link text-black-c text-over" href="./index.html">Home</a>
+                                </li>
+                                <li class="nav-item">
+                                  <a class="nav-link text-black-c text-over text-active" href="./about.html">About Us</a>
+                                </li>
```
Comment (dasileker) [inline about.html:48]: - [ ] Please add the other links even if they will not work to make your page look similar to the design.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                    <a href="#">
+                        <img src="./images/GAS.png" width="200" alt="image_partner"/>
+                    </a>
+                </div>
+                <div class="col text-center pt-md-5">
+                    <a href="google.com">
+                        <img src="./images/google.png" class="photo" alt="image_partner"/>
+                    </a>
+                    
+                </div>
```
Comment (dasileker) [inline about.html]: - [ ] Please fix the view for the images they are too big, please make them more convenient for view.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @azy64,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### **To Highlight**
- The PR is really nice.
- The linters are passing.
- The commit are good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- HTML5
+- CSS3
+- CSS GRID AND FLEXBOX
+- LINTERS FOR HTML/CSS
+- JAVASCRIPT
+
+## Live Demo
+
+[Live Demo Link](https://azy64.github.io/Portfolio/)
```
Comment (emmanuellekamwa) [inline README.md]: - [ ] Hello. Please fix your live demo link. It brings us to your Portfolio instead of your capstone project.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @azy64 👋🏽
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @azy64 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**